ws-nextjs-app/pages/[service]/av-embeds/AvEmbedsPageLayout.tsx

```diff
@@ -1,35 +1,18 @@
 /** @jsx jsx */
-import { jsx } from '@emotion/react';
+import { Global, jsx } from '@emotion/react';
 import MediaLoader from '#app/components/MediaLoader';
 import { AvEmbedsPageProps } from './types';
+import styles from './styles';
 
-const AvEmbedsPageLayout = ({ pageData }: AvEmbedsPageProps) => {
-  return (
-    <div style={{ maxWidth: 600 }}>
-      <h1>
-        AV Embeds -{' '}
-        {pageData?.output?.isSyndicationRoute
-          ? 'Syndication'
-          : 'Non-Syndication'}
-      </h1>
-      <p>Input:</p>
-      <pre>{JSON.stringify(pageData?.input, null, 2)}</pre>
-      <p>Output:</p>
-      <pre>{JSON.stringify(pageData?.output, null, 2)}</pre>
-      {pageData?.mediaBlock && pageData?.mediaBlock?.length > 0 && (
-        <>
-          <p>Video Block:</p>
-          <MediaLoader
-            blocks={pageData?.mediaBlock}
-            css={{
-              paddingBottom: 0,
-              '> div': { aspectRatio: 'auto' },
-            }}
-          />
-        </>
-      )}
-    </div>
-  );
-};
+const AvEmbedsPageLayout = ({ pageData }: AvEmbedsPageProps) => (
+  <>
+    <Global styles={styles.global} />
+    {pageData?.mediaBlock && pageData?.mediaBlock?.length > 0 && (
+      <div data-testid="avembeds-mediaplayer">
+        <MediaLoader blocks={pageData?.mediaBlock} css={styles.mediaPlayer} />
+      </div>
```

Comment:
If no mediaBlock exists in the data, would this mean that we render nothing? Would it be better/is it possible to display a 500/404 instead? What do we currently do?

Reply:
I'll remove these checks as the BFF will return a 404 if the media asset cannot be found.

Reply:
And that would mean that it would 404 a bit higher up the chain (Belfrage?), instead of us needing to do anything in Simorgh, right?

Reply:
404s would reach us as they aren't cached upstream. Simorgh would then render the translated 404 page, but that logic would be handled in these areas:
https://github.com/bbc/simorgh/blob/766aba92b092d578dc021f622d06e9b0c33254c1/ws-nextjs-app/pages/%5Bservice%5D/av-embeds/handleAvRoute.tsx#L57
https://github.com/bbc/simorgh/blob/766aba92b092d578dc021f622d06e9b0c33254c1/ws-nextjs-app/pages/%5Bservice%5D/av-embeds/handleAvRoute.tsx#L74
https://github.com/bbc/simorgh/blob/dd9bd59167a2cf9a2da914b3fd6397a6d1c0da6f/ws-nextjs-app/pages/_app.page.tsx#L76-L81